pkg/kubelet/cm/dra/plugin/noderesources.go

```diff
@@ -441,13 +451,27 @@ func (c *nodeResourcesController) sync(ctx context.Context, driverName string) e
 			continue
 		}
 
+		node, err := c.getNode()
+		if err != nil {
+			logger.Error(err, "Creating node resource slice failed")
```

Comment:
Please don't log errors when they are returned to the caller. The caller decides whether and how errors need to be logged.

Reply:
done, thanks for pointing out